### DISCUSSION TITLE
Save created nodes! And get them from a node list!

### DISCUSSION THREAD

**Discussion by wolzjohannes** [nxt-dev/nxt_editor#141] [Ideas]
Hey guys first of all! NXT is awesome! And i would like to use it more in our production! But a lots of my coworkers and myself would love to save nodes into a node list! So that we can use it later only hitting tab on the keyboard in the nodegraph! Like mayas node editor does! Is something like this possible and we did not find it?? Thx in advance for your reply!

**Comment by MichaelAldrich** [comment]
👋 Hi wolz, This idea doesn't exist today. The premise of this pattern has bounced around a bit, and could still come in a future release

The current designed workflow for sharing nodes between people, and saving them for future use, is to put them on a layer. Typically we name these layers with name that describes what they contain. For instance if you made a layer with several nodes that spoke to shotgun, maybe it would be called `shotgun_utils.nxt`. Once the nodes are saved onto that layer, anyone can reference that layer within theirs and use those nodes. This is similar to a maya reference, where if the source file is updated, everyone referencing it receives the updates.

**Reply by wolzjohannes** [reply]
Hey Michael thx for your quick reply! Yeah i saw that workflow with openRig! I
And it works fine! But anyway maybe somedays you will add that feature! That would make NXT more awesome then it is! THX
